README.md

```diff
@@ -178,18 +191,42 @@ Use the available commands for more info on the functionality available.
 
 #### Credentials error
 
-`error creating group commands for env: develop: error fetching ec2: {"develop" "development"}: NoCredentialProviders: no valid providers in chain. Deprecated.`
+1. If sandbox/prod/staging are not in the dp cli output try unsetting `AWS_REGION` and `AWS_DEFAULT_REGION`
+
+1. `SSOProviderInvalidToken: the SSO session has expired or is invalid`
+
+If you see the above error, you need to re-authenticate with sign in information
+
+1. `error fetching ec2: {Name:sandbox Profile:dp-sandbox User:ubuntu Tag:awsb CI:false ExtraPorts:{Bastion:[] Publishing:[] Web:[]}}: MissingRegion: could not find region configuration`
+
+check that you have the correct AWS profile names in your `~/.aws/config` file (dp-sandbox, dp-staging, dp-prod, dp-ci)
+
+Sample of file `~/.aws/config` for sandbox profile:
+
+[profile dp-sandbox]
+sso_start_url = https://ons.awsapps.com/start
+sso_account_id = 1234556253 #replace this with correct account id
+sso_role_name = AdministratorAccess
+sso_region = eu-west-2
+region = eu-west-2
+
+1. `Error: no security groups matching environment: "sandbox" with name "sandbox - bastion"`
+
+check  ~/.aws/credentials and remove any profile information added for dp-sandbox/dp-staging/dp-prod as this is not needed
```

Comment:
I don't think the numbering is worked as expected if you look at the rendered output.